pkg/crypto/api.go

```diff
@@ -42,7 +42,8 @@ type Crypto interface {
 
 	// WrapKey will execute key wrapping of cek using apu, apv and recipient public key 'recPubKey'.
 	// 'opts' allows setting the option sender key handle using WithSender() option. It allows ECDH-1PU key wrapping
-	// (aka Authcrypt). The absence of this option uses ECDH-ES key wrapping (aka Anoncrypt).
+	// (aka Authcrypt). The absence of this option uses ECDH-ES key wrapping (aka Anoncrypt). Another option that can
+	// be used is WithXC20PKW() to instruct the WrapKey to use XC20P key wrapping instead of the default A256GCM.
```

Comment:
```suggestion
	// be used is WithXC20PKW() to encrypt the ephemeral key (`epk`) with XC20P instead of the default A256GCM.
```

Reply:
to be more specific: it wrap the cek using XC20P instead of A256GCM, the ephemeral key (epk) is used to along with the recipient key (and sender for 1PU) to derived the wrapping key (kek) used to wrap the cek.

WithXC20PKW() option is for the wrapping (encryption) alg used to wrap the cek, not the epk